cli/ops/net.rs

```diff
@@ -317,10 +513,43 @@ impl TcpListenerResource {
   }
 }
 
+#[cfg(unix)]
+struct UnixDatagramResource {
+  socket: UnixDatagram,
+  local_addr: unix::net::SocketAddr,
+}
+
 struct UdpSocketResource {
   socket: UdpSocket,
 }
 
+#[derive(Deserialize)]
+struct IpListenArgs {
+  hostname: String,
+  port: u16,
+}
+
+#[cfg(unix)]
+#[derive(Deserialize)]
+struct UnixListenArgs {
+  address: String,
+}
+
+#[derive(Deserialize)]
+#[serde(untagged)]
+enum ArgsEnum {
+  Ip(IpListenArgs),
+  #[cfg(unix)]
+  Unix(UnixListenArgs),
+}
+
+#[derive(Deserialize)]
+struct ListenArgs {
+  transport: String,
+  #[serde(flatten)]
+  transport_args: ArgsEnum,
+}
+
```

Comment:
Given all the `#[cfg(unix)]` attributes in this file, would it make sense to add a new module`cli/ops/net_unix.rs` (or however you want to name it) which lifts all the `#[cfg(unix)]` to `cli/ops/mod.rs` ?